model/ftn/w3iorsmd.ftn

```diff
@@ -609,7 +604,7 @@
 !/MPI                          IF ( IP .EQ. NAPRST ) THEN
 !/MPI                              WRITEBUFF(1:NSPEC) = VA(1:NSPEC,JSEA)
 !/MPI                            ELSE
-!/MPI                              JSEA   = JSEA - 2*((IB-1)/2)*RSBLKS
+!/MPI                              JSEA = 1 + MOD(IB-1,2)*RSBLKS
```

Comment:
Do the bug fixes here warrant changing the date for this file?

Reply:
I was going to say no, as it as it doesn't actually change the format of the file... but I suppose it could change how an earlier file was read in w.r.t. JSEA? @ukmo-juan-castillo - can you confirm?